examples/safer_cluster_iap_bastion/kms.tf

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+data "google_project" "project" {}
+
+locals {
+  gke_sa = "serviceAccount:service-${data.google_project.project.number}@container-engine-robot.iam.gserviceaccount.com"
+}
+
```

Comment:
We can generate this using [service identities](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/project_services/variables.tf#L34) from the project services module in apis.tf and use the `enabled_api_identities` output to retrieve the sa email.

Reply:
Hi @bharathkkb Thanks for the response. (activate_api_identities)[https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/project_services/variables.tf#L34] required the api and the role. I can pass the api but what should be the role I need to pass ? Because this service account needs the KMS encrypter/Decrypter role which is taken care at the KMS Module
Something like. this

```
 encrypters = [
   "serviceAccount:${module.enabled_google_apis.enabled_api_identities["container.googleapis.com"]}",
  ]
  decrypters = [
  "serviceAccount:${module.enabled_google_apis.enabled_api_identities["container.googleapis.com"]}",
  ]
  
  activate_api_identities     = [{
    api = "container.googleapis.com",
    roles = [????],
  }]
```

Reply:
Updating the role at the time of api_identities and removed thee role from KMS module